cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -92,7 +92,7 @@ func Test_certManagerClient_getManifestObjects(t *testing.T) {
 					}
 				}
 				if !found {
-					t.Error("Expected to find cert-manager-webhook MutatingWebhookConfiguration with sideEffects=None")
+					t.Error("Expected to find cert-manager-webhook MutatingWebhookConfiguration with sideEffects=None,admissionReviewVersions=v1beta1")
```

Comment:
nit. 
the test currently does not check admissionReviewVersions=v1beta1, so either remove from the comment or add a test for it

Reply:
Fixed